qsh/__init__.py

```diff
@@ -292,13 +291,7 @@ def read_stream_header(self):
 
         return stream_type, instrument_code
 
-    # Last values for read_frame_header()
-    last_frame_milliseconds = None
-
     def read_frame_header(self):
-        if self.last_frame_milliseconds is None:
-            self.last_frame_milliseconds = self.last_created_at_milliseconds
```

Comment:
Moved this initialization to `__init__` method. This if-check was unnecessary 999999 times out of one million.